### PR TITLE
updating dijkstra test case

### PR DESCRIPTION
**Pull request by JaCh23**

### CODE REVIEW

**Comment by JaCh23** [comment]
fixes #54 , fixes #53

File: src/main/java/seedu/traveller/ItemsList.java [hunk 1]
```diff
@@ -26,6 +27,24 @@ public void addItem(Item item) {
         items.add(item);
     }
 
+    public void editItem(int i, Item item) {
```
Comment (Uxinnn) [inline src/main/java/seedu/traveller/ItemsList.java:30]: dont use i

File: src/main/java/seedu/traveller/ItemsList.java [hunk 2]
```diff
@@ -26,6 +27,24 @@ public void addItem(Item item) {
...
+        logger.log(Level.INFO, "Editing an item to itemsList");
+        items.add(item);
+        Collections.swap(items, i, getSize() - 1);
+        items.remove(getSize() - 1);
+    }
+
+    public ArrayList<Item> searchItem(String s) {
```
Comment (Uxinnn) [inline src/main/java/seedu/traveller/ItemsList.java:37]: dont use s

File: src/main/java/seedu/traveller/ItemsList.java [hunk 3]
```diff
@@ -26,6 +27,24 @@ public void addItem(Item item) {
...
+        ArrayList<Item> ans = new ArrayList<>();
```
Comment (Uxinnn) [inline src/main/java/seedu/traveller/ItemsList.java:38]: ans is abit iffy, can change if u can think of a better variable name

**Pull request merged**